File: Lab/python/labcycle1.1/leapYear.py
sYear = int(input("Enter start year : "))
eYear = int(input("Enter end year : "))
print(f"Leap years between {sYear} and {eYear} are : ")

while sYear<=eYear:
	if sYear%4==0 and (sYear%400==0 or sYear%100!=0):
		print(f"{sYear}")
	sYear = sYear+1

File: lab_exam/sample/java/leapYear.java
import java.util.*;

public class leapYear {
    public static void main(String[] args) {
        int s, e;
        Scanner inp = new Scanner(System.in);
        System.out.println("Enter starting year : ");
        s = inp.nextInt();
        System.out.println("Enter ending year : ");
        e = inp.nextInt();
        System.out.println("Leap years between " + s + " and " + e + " are : ");
        for (int i = s; i <= e; i++) {
            if ((i % 100 != 0 || i % 400 == 0) && i % 4 == 0) {
                System.out.println(i);
            }
        }
    }
}
